### ISSUE TITLE
Click on empty location fire error

### ISSUE THREAD

**Issue by jkelany** [hanafnafs/angular-hijri-gregorian-datepicker#3]
Click on empty location on the calendar grid 
![image](https://github.com/user-attachments/assets/b6de4230-0f18-4fb7-b75b-8f92db170601)

It will throw error: 

```
Cannot read properties of undefined (reading 'gD') TypeError: Cannot read properties of undefined (reading 'gD')
    at Ee.onDayClicked (angular-hijri-gregorian-datepicker.mjs:27829:17)
    at angular-hijri-gregorian-datepicker.mjs:27923:4903
    at Y9 (core.mjs:26657:16)
    at s (core.mjs:26691:22)
    at HTMLDivElement.<anonymous> (platform-browser.mjs:675:19)
    at I.invokeTask (zone.js:402:31)
    at Object.onInvokeTask (core.mjs:7228:33)
    at I.invokeTask (zone.js:401:60)
    at oe.runTask (zone.js:173:47)
    at j.invokeTask [as invoke] (zone.js:483:34)
```

**Comment by hanafnafs** [comment]
@jkelany Thank you for reporting the issue, will be fixed in the next release this weekend!

**Comment by imadkhan4** [comment]
Fixed on version @1.0.5, issue will be closed.
